app/forms/user_form.rb

```diff
@@ -2,11 +2,65 @@
 
 ##
 # Form object for creating or updating a User
+require 'digest/sha1'
+
 class UserForm
-  include AuthenticationForm
-  include Auditor
+  include ActiveModel::Model
+  include FormObject
```

Comment:
Something for another day but I think I would like to remove this at some point and revert to it being much simpler.